File: Swift-C++-IR-Comparison/test/structsAndClasses/Swift/Structs/StructInFunctionCall.swift
@main
struct MyMain {

    static func main() {
        let res = Resolution(width: 1920, height: 1080)
        let s = toString(res)
    }
}

func toString(_ res: Resolution) -> String {
    let width = String(res.width)
    let height = String(res.height)
    return "The resolution is \(width)x\(height)"
}

struct Resolution {
    var width = 0
    var height = 0
}

File: AutoTranspiledTests/Swift-to-Cpp/try-0/StructInFunctionCall.cpp

#include <iostream>
#include <string>

struct Resolution {
    int width;
    int height;
};

std::string toString(Resolution res) {
    std::string width = std::to_string(res.width);
    std::string height = std::to_string(res.height);
    return "The resolution is " + width + "x" + height;
}

int main() {
    Resolution res = {1920, 1080};
    std::cout << toString(res);
    return 0;
}